docs/get-started/create-app.md

```diff
@@ -1,34 +1,175 @@
+
+Now run it and observe the output of the server and client:
+
+````{tab} Run server
+
+```shell
+jina deployment --uses deployment.yml
+```
+
+```shell
+
+──── 🎉 Deployment is ready to serve! ────
+╭────────────── 🔗 Endpoint ───────────────╮
+│  ⛓     Protocol                    GRPC  │
+│  🏠       Local           0.0.0.0:54321  │
+│  🔒     Private    192.168.200.56:54321  │
+│  🌍      Public    81.223.121.124:54321  │
+╰──────────────────────────────────────────╯
+```
+````
+
+````{tab} Run client
+```bash
+python client.py
+```
+
+```shell
+['hello, world!', 'goodbye, world!']
+```
+````
+
+## Add logic
+
+You can use any Python library in {class}`~jina.Executor`. For example, add `pytorch` to `executor1/requirements.txt` and crunch some numbers. 
+
+In `executor.py`, add another endpoint `/get-tensor` as follows:
+
+```{code-block} python
+---
+emphasize-lines: 13-16
+---
+import numpy as np
+import torch
+
+from jina import Executor, requests, DocumentArray
+
+
+class MyExecutor(Executor):
+    @requests
+    def foo(self, docs: DocumentArray, **kwargs):
+        docs[0].text = 'hello, world!'
+        docs[1].text = 'goodbye, world!'
+
+    @requests(on='/crunch-numbers')
+    def bar(self, docs: DocumentArray, **kwargs):
+        for doc in docs:
+            doc.tensor = torch.tensor(np.random.random([10, 2]))
+```
+
+Kill the last server with `Ctrl-C` and restart the server with `jina flow --uses deployment.yml`.
+
+## Call `/crunch-number` endpoint
+
+Modify `client.py` to call the `/crunch-numbers` endpoint:
+
+```python
+from jina import Client, DocumentArray
+
+if __name__ == '__main__':
+    c = Client(host='grpcs://1655d050ad.wolf.jina.ai')
+    da = c.post('/crunch-numbers', DocumentArray.empty(2))
+    print(da.tensors)
+```
+
+After you save that, you can run your new client:
+
+```bash
+python client.py
+```
+
+```text
+tensor([[[0.9594, 0.9373],
+         [0.4729, 0.2012],
+         [0.7907, 0.3546],
+         [0.6961, 0.7463],
+         [0.3487, 0.7837],
+         [0.7825, 0.0556],
+         [0.3296, 0.2153],
+         [0.2207, 0.0220],
+         [0.9547, 0.9519],
+         [0.6703, 0.4601]],
+
+        [[0.9684, 0.6781],
+         [0.7906, 0.8454],
+         [0.2136, 0.9147],
+         [0.3999, 0.7443],
+         [0.2564, 0.0629],
+         [0.4713, 0.1018],
+         [0.3626, 0.0963],
+         [0.7562, 0.2183],
+         [0.9239, 0.3294],
+         [0.2457, 0.9189]]], dtype=torch.float64)
+```
+
+## Deploy to JCloud
+
+JCloud offers free CPU and GPU instances to host Jina project.
+
+```admonition
+:class: important
+At present, JCloud is only available for Flows. We are currently working on supporting Deployments.
+```
+
+You've just finished your first toy Jina project, congratulations! You can now start your own project.
+
+````
+
+````{tab} Flow
+
+To create a flow project with Jina CLI:
```

Comment:
"Flow" should have capital first letter. Also "Deployment" - can you check throughout the doc that they're capitalized correctly?

```suggestion
To create a Flow project with Jina CLI:
```

Reply:
Alright, Thank you. Will get to that @alexcg1